Source/WebCore/animation/CSSPropertyAnimation.cpp

```diff
@@ -163,13 +163,21 @@ static inline std::unique_ptr<ShadowData> blendFunc(const ShadowData* from, cons
 {
     ASSERT(from && to);
     ASSERT(from->style() == to->style());
+    
+    auto resolveColor = [&] (const StyleColor& color) {
+        if (!context.client)
+            return Color { };
+
+        auto& renderStyle = context.client->currentStyle();
+        return renderStyle.colorResolvingCurrentColor(color);
+    };
```

Comment:
Maybe we can pass in the RenderStyle from `PropertyWrapperShadow::blend()`?

Reply:
Extracing the `RenderStyle` from `context` in the caller and passing it instead of doing the extraction here ?

Reply:
No, passing it down from `PropertyWrapperShadow::blend()` where the from/to `RenderStyle` are readily available.

Reply:
Indeed, those are the correct `RenderStyle`, not the one from `context`. Fixed